deps/rabbit/src/rabbit_mirror_queue_misc.erl

```diff
@@ -201,12 +201,13 @@ drop_mirror(QName, MirrorNode) ->
     case rabbit_amqqueue:lookup(QName) of
         {ok, Q} when ?is_amqqueue(Q) ->
             Name = amqqueue:get_name(Q),
-            QPid = amqqueue:get_pid(Q),
-            SPids = amqqueue:get_slave_pids(Q),
-            case [Pid || Pid <- [QPid | SPids], node(Pid) =:= MirrorNode] of
+            PrimaryPid = amqqueue:get_pid(Q),
```

Comment:
John O'Hara - inventor of AMQP - will be sad to see the demise of QPid - AMQP's first implementation - from the RabbitMQ codebase

Reply:
Hahaha. Definitely a good one 🤓